RiotSharp/RiotApi.cs

```diff
@@ -33,9 +34,7 @@ public class RiotApi : IRiotApi
 
         private const string NamesUrl = "/{0}/name";
         private const string MasteriesUrl = "/lol/platform/v3/masteries/by-summoner/{0}";
```

Comment:
Should we create a RootUrl for Masteries + Runes + Champions ? because it's the same.

Reply:
yup, good idea :+1: , let's refactor `/lol/platform/v3` into its own variable

Reply:
I would be careful with this. Riot might change some urls to something like `/lol/platform/v3.1`, but not all.

Reply:
yup true as well